sourcegraph/learn
# Add two Sourcegraph videos

### What should this PR do?
Adds two videos, resolving DEVED-130 and DEVED-131.

### Why are we making this change?
Product-related educational video resources.

### What are the acceptance criteria? 
Ensure that embedded YouTube videos are correct and play. Ensure that tutorials are formatted correctly. 

### How should this PR be tested?
Check out branch locally and check the frontend of the site, or else check the deployment preview. 

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
